SpatialGDK/Documentation/content/simulated-players.md

```diff
@@ -122,5 +122,6 @@ Logs for simulated players are stored as [raw logs](https://docs.improbable.io/r
 
 <br/>
 <br/>------------<br/>
+_2019-11-28 Page updated without editorial review: added automatic sim players launching using batch scripts._<br/>
```

Comment:
"sim players" is not a term we use.
"automatic sim players launching" as a noun is quite hard to follow.


```suggestion
_2019-11-28 Page updated without editorial review: added batch scripts to launch simulated players._<br/>
```